.github/shared/test/changed-files.test.js

```diff
@@ -92,6 +94,11 @@ describe("changedFiles", () => {
     expect(files.filter(specification)).toEqual(expected);
   });
 
+  it("filter:typespec", () => {
+    const expected = ["specification/contosowidgetmanager/Contoso.Management/main.tsp"];
```

Comment:
shouldn't `expected` include `tspconfig.yaml`?

Reply:
You know, it absolutely should. Why is this test passing as-is. Will fix before merge.